addons/im_livechat/static/src/messaging/entity/messaging_initializer/messaging_initializer.js

```diff
@@ -0,0 +1,32 @@
+odoo.define('im_livechat.messaging.entity.MessagingInitializer', function (require) {
+'use strict';
+
+const { registerInstancePatchEntity } = require('mail.messaging.entity.core');
+
+registerInstancePatchEntity('MessagingInitializer', 'im_livechat.messaging.entity.MessagingInitializer', {
+
+    //----------------------------------------------------------------------
+    // Private
+    //----------------------------------------------------------------------
+
+    /**
+     * @override
+     * @param {Object[]} [param0.channel_slots.channel_livechat=[]]
+     */
+    _initChannels(initMessagingData) {
+        this._super(initMessagingData);
+        const { channel_livechat = [] } = initMessagingData.channel_slots;
+        for (const data of channel_livechat) {
+            this.env.entities.Thread.insert(Object.assign(
+                {},
+                this.env.entities.Thread.convertData(data),
+                {
+                    isPinned: true,
```

Comment:
should be default value, not overriding if it's set in convertData

Reply:
process dependent to set it, and `isPinned: false` by default. `convertData` cannot deduce this from server data.